packages/ui/src/i18n/dictionaries/authenticator/es.ts

```diff
@@ -14,10 +15,12 @@ export const esDict = {
   'Confirmation Code': 'Código de confirmación',
   Confirming: 'Confirmando',
   'Create a new account': 'Crear una cuenta nueva',
+  'Create Account': 'Crear cuenta',
   'Creating Account': 'Creando cuenta',
   Email: 'Email',
+  'Enter your email': 'Email',
   'Forgot Password': 'Olvidé mi contraseña',
-  'Incorrect username or password': 'Nombre de usuario o contraseña incorrecta',
+  'Incorrect username or password.': 'Nombre de usuario o contraseña incorrecta',
```

Comment:
Why add the period here? Won't this break the translation?

Reply:
Currently, english version has that period
![image](https://user-images.githubusercontent.com/20952412/178886214-7bfdf8fd-a770-4101-b14c-7d468d5236e2.png)
this translation is not working